.gitignore

```diff
@@ -7,7 +7,7 @@ dist
 build
 _build
 distribute-*
-env
+venv*/
```

Comment:
I was going to say that the docs would need to be updated to match, but apparently the how-to instructions already use `venv` and not `env`! So thank you for fixing this to match the docs 🙂